app/api-requests/rpa-api/base.js

```diff
@@ -1,11 +1,13 @@
 const wreck = require('@hapi/wreck')
 const session = require('../../session')
 const { tokens } = require('../../session/keys')
+const config = require('../../config')
 
 const get = async (hostname, url, request, headers = {}) => {
   const token = session.getToken(request, tokens.accessToken)
-  headers.Authorization = token
-  console.log(`${new Date().toISOString()} Request Message to RPA: ${JSON.stringify(`${hostname}${url}`)}`)
+  headers['X-Forwarded-Authorization'] = token

```

Comment:
Could we create a little constants file for APIM headers please?

Reply:
Code updated - constant file added and used by base.js